migration/v0.31/prepare/07-move-subdomains.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
you will need to move this scripts to v0.33
maybe also merge them, as I do not see a reason to keep the separately

Reply:
done